### DISCUSSION TITLE
Weatherian - multi-platform multi-model meteogram app

### DISCUSSION THREAD

**Discussion by moneytoo** [open-meteo/open-meteo#301] [Show and tell]
Hello,

please let me introduce a weather app I created. 

A little history first. In 2018 I shortly switched to using iPhone SE from Android and was suddenly missing my favorite weather forecast apps (using the Czech model Aladin from CHMU). Not only that - most weather apps spread forecast information across many screens or tabs and require just too many taps to find out forecast data for at least the next 2 to 3 days. I ended app creating a web app showing a meteogram where I had to reverse engineer (and proxy on a server) some forecast APIs. Unfortunately, this prevented me to spread the word about this app outside a small community, which liked it. I found out that adding multiple models to a single chart is extremely useful to me (especially when travelling) so I can be better prepared as well as know to chances and possible extremes. Another thing that ended up being important is the scale and intervals of the chart - one must keep it as stable as possible to make it readable like a clock (as lots of people prefer analog clock over digital).

<img src="https://user-images.githubusercontent.com/1071643/231783220-5e6c436b-ac9b-425a-8d47-f1ce4dce1a2e.PNG" height="320">

So here we are. I always wanted to create the ultimate weather app (based on my experience) but was stopped by various licencing terms. Here's the big thanks to Open-Meteo for making things possible. I'm now able to work on my little side project and materialize an app for these needs.

![1-framed](https://user-images.githubusercontent.com/1071643/231769776-cb603877-e5b0-4b19-b59b-7a30759d0b6e.png)

You can try the app in its current state here: [Weatherian.com](https://weatherian.com/) (also available for download for most mobile as well as desktop platforms). So again, thank you Open-Meteo and @patrick-zippenfenig for the API. I'm currently using your servers but I'm keeping an eye on the user base to fire up own instance if things take off.

Best regards,
Marcel

**Comment by patrick-zippenfenig** [comment]
Really cool! Thanks for creating it! Could you create a PR and add it the Readme of this project?

Don't worry about the number of API calls as this it clearly non-commercial and open-source
